src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -289,6 +291,127 @@ class Cas2ApplicationTest : IntegrationTestBase() {
     }
   }
 
+  @Nested
+  inner class GetToIndexUsingIsSubmitted {
```

Comment:
very minor but I wonder if this could be Nested within the existing `GetToIndex` Nested? As I guess it's all testing the same function...

Reply:
I've left this as is for now, just because there is a `@BeforeEach` `setup` function for these tests.